channeldb/duplicate_payments.go

```diff
@@ -59,22 +59,22 @@ type duplicateHTLCAttemptInfo struct {
 	route route.Route
 }
 
-// fetchDuplicatePaymentStatus fetches the payment status of the payment. If the
-// payment isn't found, it will default to "StatusUnknown".
-func fetchDuplicatePaymentStatus(bucket kvdb.RBucket) PaymentStatus {
+// fetchDuplicatePaymentStatus fetches the payment status of the payment. If
```

Comment:
PR description "Depending on https://github.com/lightningnetwork/lnd/pull/7175" is self-referring.

Reply:
fixed!